frontend/src/features/admin-form/responses/FeedbackPage/issue/IssueService.tsx

```diff
@@ -0,0 +1,39 @@
+import { FormIssueDto, FormIssueMetaDto } from '~shared/types'
+
+import { ApiService } from '~services/ApiService'
+
+import { ADMIN_FORM_ENDPOINT } from '~features/admin-form/common/AdminViewFormService'
+import { IssueCsvGenerator } from '~features/admin-form/responses/FeedbackPage/utils/IssueCsvGenerator'
+
+export const getFormIssues = async (
+  formId: string,
+): Promise<FormIssueMetaDto> => {
+  return ApiService.get<FormIssueMetaDto>(
+    `${ADMIN_FORM_ENDPOINT}/${formId}/issues`,
+  ).then(({ data }) => data)
+}
+
+/**
+ * Download issues for a given form.
+ * @param formId the id of the form to download issues for
+ * @param formTitle the title of the form
+ * @param count estimated issue counts
+ * @returns a stream of feedback
+ */
+export const downloadFormIssue = async (
+  formId: string,
+  formTitle: string,
+  count: number,
+): Promise<void> => {
+  return ApiService.get<FormIssueDto[]>(
+    `${ADMIN_FORM_ENDPOINT}/${formId}/issues/download`,
+  ).then(({ data }) => {
+    const csvGenerator = new IssueCsvGenerator(count)
+
+    data.forEach((issue) => {
+      csvGenerator.addLineFromIssue(issue)
+    })
+
+    csvGenerator.triggerFileDownload(`${formTitle}-${formId}-issue.csv`)
```

Comment:
(non-blocking / can be a separate PR) - wondering if it'd be useful to add logs for this!

Reply:
yup like frontend logs! can search `datadogLogs` in the codebase :)

Reply:
wow, til, what is the diff between `datadogLogs` and the `logger` that I have been using?

Reply:
ya good qn!! `logger` is used in the backend, the logs go to AWS Cloudwatch. We have a lambda which pipes the Cloudwatch logs over to Datadog as well.

 `datadogLogs` are used in the frontend, the logs go directly to Datadog.